.github/workflows/ci.yml

```diff
@@ -182,4 +182,91 @@ jobs:
         with:
           name: MacOS CMake Test Logs
           path: build-aux/artifacts
-    
\ No newline at end of file
+
+  Windows-Autotools:
+    # Just run on latest Windows
+    runs-on: windows-latest
+    steps:
+      # Fixes line engings, which fails compilation
+      # https://github.com/actions/checkout/issues/226
+      # https://github.com/actions/checkout/issues/135
+      - name: Prepare git
+        run: git config --global core.autocrlf false
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      - name: Install Dependencies
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: UCRT64
+          update: true
+          install: >-
+            base-devel
+            mingw-w64-ucrt-x86_64-toolchain
+            git
+            autoconf
+            automake
+            mingw-w64-ucrt-x86_64-autotools
+            po4a
+
+      - name: Build with full features
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -p build
+      - name: Test with full features
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -p test -n full_features
+
+      - name: Build without encoders
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -d encoders,shared -p build
+      - name: Test without encoders
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -d encoders,shared -p test -n no_encoders
+
+      - name: Build without decoders
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -d decoders,shared -p build
+      - name: Test without decoders
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -d decoders,shared -p test -n no_decoders
+
+      - name: Build without threads
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -d threads,shared -p build
+      - name: Test without threads
+        shell: msys2 {0}
+        run: ./build-aux/ci_build.sh -b autotools -d threads,shared -p test -n no_threads
+
+      - name: Build without BCJ filters
```

Comment:
Since the Windows CI takes so long, we can remove some of these less interesting tests. Let's omit the BCJ, Delta, and SHA256 trials. The CRC64 one could still be interesting, so lets keep that one.